api/models/people.py

```diff
@@ -131,6 +147,20 @@ def __str__(self) -> str:
         role = " (admin)" if self.is_admin else ""
         return f"{super().__str__()}{role}"
 
+    @property
+    def has_enough_groups(self) -> bool:
+        """Returns `True` if coordinator has required minimum amount of groups."""
+        if self.group_set.count() >= CoordinatorGroupLimit.MIN:
+            return True
+        return False
+
+    @property
+    def has_reached_group_limit(self) -> bool:
+        """Returns `True` if coordinator has reached maximum amount of groups."""
+        if self.group_set.count() >= CoordinatorGroupLimit.MAX:
```

Comment:
Let's add related_name for this to be `groups`

Reply:
I mean instead of group_set

Reply:
Done

Reply:
I couldn't just put "groups" though, had to use `%('class')ss`